MojioSDK/Client/Core/RestClient.swift

```diff
@@ -48,6 +48,15 @@ open class ClientHeaders {
     }
 }
 
+public enum XMojioVersion: String {
+    case current = "2018-09-01"
+    case eLogBook = "2020-03-17"
```

Comment:
I'd like to have v4

Reply:
@andriimakukha I don't like the "naming" too. And I like the idea of v4.
But at the same time 
https://tmobile-us-phoenix-production-api.moj.io/ - "version":"**4.**13.0.187" - has no eLogBook support
https://telekom-de-phoenix-staging-api.moj.io/ - "**4.**14.0.1354" - has eLogBook support

Reply:
Do we use "2020-03-17" for TMUS?

Reply:
@andriimakukha No, I made that as "Brand Flag" on the app level. 
https://github.com/mojio/Phoenix-iOS/pull/4550/files

Reply:
I see that no, so for TMUS we use v3 that is "2018-09-01".

Reply:
@andriimakukha are we good to merge this? Don't want to break any functionality in doing so. Want to be sure. Thx.

Reply:
If it's urgent, I think yes but need to create tech debt ticket to align versions of API

Reply:
Ah OK. This will need to go in 3.8.8, 3.8.9 and develop.